src/definitions.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    info: {
+   
+      title: 'The Alcoding Club Application Api\'s',
```

Comment:
Can we fetch this info from package.json. Swagger may have an option to do this.